### PR TITLE
Add Astron Energy

### PR DESCRIPTION
**Pull request by DivyaB1**
Hi,
I am adding 'Astron Energy' - https://www.astronenergy.co.za/

### CODE REVIEW

File: data/brands/amenity/fuel.json [hunk 1]
```diff
@@ -25,6 +25,16 @@
...
   },
   "items": [
+    {
+     "displayName": "Astron Energy",
+     "locationSet": {"include": ["za"]},
+     "tags": {
+       "amenity": "fuel",
+       "brand": "Astron Energy",
+       "brand:wikidata": "Q120752181",
+       "name": "ASTRON ENERGY"
```
Comment (bhousel) [inline data/brands/amenity/fuel.json:35]: From their store locator page, it looks like they call themselves "Astron Energy" in most places and on maps.  


Screenshot 2023-07-24 at 9 52 51 AM

The allcaps "ASTRON ENERGY" name looks kind of weird (this is more normal in Japan but not in South Africa as far as I know).  Would you be ok with just using Title Case here?




```suggestion
       "name": "Astron Energy"
```

**Review by bhousel — COMMENTED** [review]
Just one question about the name case

File: data/brands/amenity/fuel.json (hunk 1 above)
Comment (DivyaB1) [reply data/brands/amenity/fuel.json:35]: Thanks @bhousel, for sure I'll update the name case.
Quick question about the 'name' tag - Does the name have to be what the logo of brand says or what it says on their map/store locator?
While adding 'Reel Cinemas' I got an impression it should be the logo, so used the same logic https://github.com/osmlab/name-suggestion-index/pull/8311#issuecomment-1607971655

**Comment by DivyaB1** [comment]
Thanks, I've updated the name case. Question about this tag - does it have to be what the logo says or what the store locator/maps say. After https://github.com/osmlab/name-suggestion-index/pull/8311#issuecomment-1607971655 I was under the impression 'name' tag gets what the logo has.

**Comment by Cj-Malone** [comment]
OSM name should be the fascia/sign.

But OSM has a complicated history with all caps. It's considered an error in many QA tools.

**Comment by bhousel** [comment]
> Question about this tag - does it have to be what the logo says or what the store locator/maps say. After [#8311 (comment)](https://github.com/osmlab/name-suggestion-index/pull/8311#issuecomment-1607971655) I was under the impression 'name' tag gets what the logo has.

We try to find a balance between how the brand calls themself and what name would look like in a list or on a map pin.  So we really don't use all caps names unless it's in a place that writes them normally this way (like Japan), or for a brand that is really consistent about writing their name this way (like IKEA or JYSK). We do get some people very particular about the uppercase names that will open issues about it.

See https://github.com/osmlab/name-suggestion-index/issues/6662#issuecomment-1177618297
or https://github.com/osmlab/name-suggestion-index/issues/5509#issuecomment-944450197

**Pull request merged**